src/app/static/src/js/index.js

```diff
@@ -9,7 +9,47 @@ require("treetables")(window, $);
 $(document).ready(function () {
     const isReviewer = typeof canReview !== "undefined";
     const $table = $('#reports-table');
+
+    $("th input, th select").on("click", (e) => {
+        // we don't watch the column sort event triggered when filtering
+        e.stopPropagation();
+    });
+
     $table.treeTable(options(isReviewer, reports));
+    const dt = $table.DataTable();
+
+    if (isReviewer) {
+        $.fn.dataTable.ext.search.push((settings, data) => {
+            const status = $('#status-filter').val();
+            return statusFilter(status, data);
+        });
+
+        $('#status-filter').change(() => {
+            dt.draw();
+        });
+    }
+
+    $('[data-role=standard-filter]').on('keyup', function () {
```

Comment:
Pasting from context menu works in  the old reportle too..

Reply:
I can't find an event that works for this, any ideas?? I tried `on("input change paste keyup mouseup")` and nada. Maybe it's ok if users have to hit enter after pasting from context menu

Reply:
Weird, i would have thought paste would have done it, or failing that the input event. Oh well!